test/audience.robot

```diff
@@ -30,14 +30,36 @@ Token with correct audience is accepted
     Should Contain   ${out}   404
 
 Token with correct audience in multiple option is accepted
+    [Tags]   audience
     ${uuid}   Generate UUID
     ${se_config}   Get SE config
     ${token}   Get token   scope=-s openid   opts=--aud="${se_config.endpoint} ${uuid}"
     ${url}   SE URL   audience-test-${uuid}
     ${rc}   ${out}   Curl Error   ${url}
     Should Contain   ${out}   404
 
-Token with invalid multiple audiences is rejected
+Token with invalid multiple audiences is rejected (permissive)
+    [Tags]   audience   permissive
+    Get token with invalid multiple audience
+    ${ret}   Should Match Regexp   ${out}   40[134]
+
+Token with invalid multiple audiences is rejected (strict)
+    [Tags]   audience   strict
+    ${out}   Get token with invalid multiple audience
+    ${ret}   Should Match Regexp   ${out}   401
+
+
+*** Keywords ***
+
+Get token with random audience
+    ${token}   Get token
+    ${uuid}   Generate UUID
+    ${url}   SE URL   audience-test-${uuid}
+    ${token}   Get token   scope=-s openid   opts=--aud=${uuid}
```

Comment:
Agreed. In some case to me is useful for debug to store variables which we don't use later on since their value is shown in the log file (as `INFO`) anyhow. Anyway, the `Get token [args]` keyword itself already stores the token in a logged environment variable, thus I totally agree that is a useless repetition.